blueprints/ember-responsive/files/app/breakpoints.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  mobile:  '(max-width: 768px)',
+  tablet:  '(min-width: 769px) and (max-width: 992px)',
+  desktop: '(min-width: 993px) and (max-width: 1200px)',
+  jumbo:   '(min-width: 1201px)'
+};
```

Comment:
Seemed like a sane default to place at runtime (since users likely copy/paste from the README anyway).